dashboard/db/migrate/20160407173756_add_course_unit_to_learning_modules.rb

```diff
@@ -0,0 +1,25 @@
+class AddCourseUnitToLearningModules < ActiveRecord::Migration
+  def up
+    add_reference :plc_learning_modules, :plc_course_unit, index: true, foreign_key: true
+
+    # Create a default Plc::CourseUnit to own all orphaned Learning Modules
+    default_course = Plc::Course.find_or_create_by! name: 'Default Course'
```

Comment:
I'd rather add it to a course with a specific name so running the migration forwards and backwards a number of times doesn't create a bunch of new Courses.